[ADD] POS: configure and use the ship later feature (#7406)
odoo/documentation: content/applications/sales/point_of_sale/shop/ship_later.rst

==========
Ship later
==========

The **Ship Later** feature allows you to sell products and schedule delivery at a later date. This
feature comes in handy when a product is out of stock or so voluminous that it requires to be

> shipped.

Author: I've added something in that sense

Configuration
=============

> Go to the POS settings, scroll down to the :guilabel:`Inventory` section, and enable :guilabel:`Ship

Author: hahahha merci x')

Later`. Once enabled, you can choose the the location from where the products are shipped by
completing the :guilabel:`Warehouse` field, define a specific route, and the shipping policy. Select
:guilabel:`As soon
as possible` if the products can be delivered separately or :guilabel:`When all products are ready`
to ship all the products in one delivery.

.. image:: ship_later/settings.png
   :scale: 75%

.. seealso::
   - :doc:`../../../inventory_and_mrp/inventory/shipping/setup/delivery_method`
   - :doc:`../../../inventory_and_mrp/inventory/management/warehouses/warehouses_locations`

Practical application
=====================

#. :ref:`Open a session <pos/session-start>` and make a sale.
#. On the payment screen, set a customer and select :guilabel:`Ship Later`.
#. On the popup window, set a shipping date and click :guilabel:`Confirm` to proceed to payment.

.. image:: ship_later/payment.png
   :scale: 75%

The system instantly creates a delivery order from the warehouse to the shipping address.

.. Note::

>    The selected customer must have referenced an address in the system for products to be shipped.

Reviewer: What happens if they don't?

==========
Ship later
==========

> The **Ship Later** feature allows you to sell products and schedule delivery at a later date. This
> feature comes in handy when a product is out of stock, so voluminous that it requires to be

Reviewer 2: ```suggestion
The **Ship Later** feature allows you to sell products and schedule delivery at a later date. It is useful, for example, when a product is out of stock or so voluminous that it requires to be
```

shipped, or if, for any reason, the customer needs their order to be shipped later, etc.

Configuration
=============

:ref:`Go to the POS settings <configuration/settings>`, scroll down to the :guilabel:`Inventory`
section, and enable :guilabel:`Ship Later`.

.. image:: ship_later/settings.png
   :scale: 85%

Once activated, you can:

> - Choose the location from where the products are shipped by completing the :guilabel:`Warehouse`
>   field.

Reviewer 2: ```suggestion
- Choose the location from where the products are shipped by selecting a :guilabel:`Warehouse`.
```
since you don't type anything but just select it?

- Define a :guilabel:`Specific route`, or leave this field empty to use the default route.
- Define the :guilabel:`Shipping Policy`; select :guilabel:`As soon as possible` if the products
  can be delivered separately or :guilabel:`When all products are ready` to ship all the products at

>   once.

Reviewer 2: You didn't think I'd leave a review without one of my famous dumb questions, did you? :smile: so here it is: what defines "ready" exactly?

Author: When all the products are available -
For example: the time to be delivered differs, you only send the products in one delivery when they have all arrived.
